control-plane-operator/controllers/hostedcontrolplane/etcd/reconcile_test.go

```diff
@@ -0,0 +1,51 @@
+package etcd
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/support/config"
+)
+
+func TestBuildEtcdInitContainer(t *testing.T) {
+	tests := []struct {
+		name   string
+		params EtcdParams
+	}{
+		{
+			name: "single replica",
```

Comment:
nit: usually when "input" it should "action" is usually nice to understand the function expectation for the test..

Reply:
Added some more detail to the test names